kernel/init.c

```diff
@@ -121,7 +121,7 @@ void z_bss_zero(void)
 #endif	/* CONFIG_CODE_DATA_RELOCATION */
 #ifdef CONFIG_COVERAGE_GCOV
 	(void)memset(&__gcov_bss_start, 0,
-		 ((uint32_t) &__gcov_bss_end - (uint32_t) &__gcov_bss_start));
+		 (uintptr_t) &__gcov_bss_end - (uintptr_t) &__gcov_bss_start);
```

Comment:
Was the protective outer parenthesis violating coding guidelines? jw, I like to use them but may be wrong to.

Reply:
@jenmwms You are totally right, I omitted this, and it might be necessary for some safety coding guideline, thanks!